packages/atomic/src/components/common/result-children/result-children-common.tsx

```diff
@@ -128,7 +128,9 @@ export class ResultChildrenCommon {
     }
 
     if (!collection.moreResultsAvailable && !this.hasChildren) {
-      return this.renderNoResult();
+      return this.props.getNoResultText()?.trim().length
```

Comment:
Perhaps it would be worthwhile to add a Cypress test for this behaviour? 🙂

Reply:
wise words! I see we don't have any for the initial display of this message either so i'll add a few.